example_test.go

```diff
@@ -252,3 +280,34 @@ func ExampleCopy_localToRemote() {
 	// Output:
 	// sha256:6f2590d54af17afaca41a6243e3c01b368117d24b32e7581a6dee1d856dd3c4b
 }
+
+// Example_copyArtifactManifestRemoteToLocal gives an example of copying
+// an artifact manifest from remote location to local.
+func Example_copyArtifactManifestRemoteToLocal() {
+	src, err := remote.NewRepository(fmt.Sprintf("%s/%s", remoteHost, "source"))
+	if err != nil {
+		panic(err)
+	}
+	dst := memory.New()
+	ctx := context.Background()
+
+	exampleDigest := "sha256:f9308ac4616a808210c12d049b4eb684754a5acf2c3c8d353a9fa2b3c47c274a"
+	exampleDescriptor, err := src.Resolve(ctx, exampleDigest)
+	if err != nil {
+		panic(err)
+	}
+	err = oras.CopyGraph(ctx, src, dst, exampleDescriptor, oras.DefaultCopyGraphOptions)
+	if err != nil {
+		panic(err)
+	}
+
+	// verify that the content of dst is described by exampleSignatureManifestDescriptor
```

Comment:
nit: verify that the artifact manifest described by the descriptor exists in dst